src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -1442,6 +1448,16 @@ void initializeLogger()
 #endif  // AXOM_USE_MPI
 
   slic::addStreamToAllMsgLevels(logStream);
+
+  conduit::utils::set_error_handler([](auto& msg, auto& file, int line) {
+    slic::logErrorMessage(msg, file, line);
+  });
+  conduit::utils::set_warning_handler([](auto& msg, auto& file, int line) {
+    slic::logWarningMessage(msg, file, line);
+  });
+  conduit::utils::set_info_handler([](auto& msg, auto& file, int line) {
+    slic::logMessage(slic::message::Info, msg, file, line);
+  });
```

Comment:
@gunney1 -- These callbacks allowed me to see the messages/warnings/errors form conduit